src/services/updateDesignSet.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @copyright Copyright (c) 2019 John Molakvoæ <skjnldsv@protonmail.com>
```

Comment:
This is your code :)

Reply:
Part of it is still yours, just moved in from another file. But I added another line 👍

Reply:
The file is yours, I'm just an author :)
One copyright per file ;)

Reply:
Updated